eZ/Bundle/EzPublishCoreBundle/Resources/config/services.yml

```diff
@@ -118,3 +121,11 @@ services:
             - [setFragmentPath, [%fragment.path%]]
             - [setSiteAccess, [@ezpublish.siteaccess]]
         abstract: true
+
+    ezpublish.converter.repository:
```

Comment:
I kept it simple for now, do you guys think I should inject the repository or we should do it if we extend it later ?